pkg/metrics/metrics.go

```diff
@@ -225,21 +225,30 @@ var (
 		Help:      "Total number of successful policy regenerations",
 	})
 
+	// Deprecated: this metric will be removed in Cilium 1.6
 	// PolicyRegenerationTime is the total time taken to generate policies
 	PolicyRegenerationTime = prometheus.NewCounter(prometheus.CounterOpts{
 		Namespace: Namespace,
 		Name:      "policy_regeneration_seconds_total",
-		Help:      "Total sum of successful policy regeneration times",
+		Help:      "Total sum of successful policy regeneration times (Deprecated)",
 	})
 
+	// Deprecated: this metric will be removed in Cilium 1.6
```

Comment:
Can you create an issue to track the removal of these in Cilium v1.6?

Reply:
Sure, #7440